sirepo/feature_config.py

```diff
@@ -174,6 +174,22 @@ def b(msg, dev=False):
             "codes that contain proprietary information and authorization to use is granted through oauth",
         ),
         raydata=dict(
+            catalog_names=(frozenset(), set, "list of catalog names to monitor"),
+            concurrent_analyses=(
```

Comment:
I think these all belong in pkcli.raydata (or sirepo/raydata see below) since that is the only place they are used.  Raydata is unlike other codes so configuring it differently I think is ok.

scan_monitor_url can stay here though since that is used in sirepo.template.raydata which works like other codes.

Reply:
e799cdc8671eaf3a2fa69713cd0682b70a4ea7bb